docs/hardware/devices/heltec-automation/sensor/index.mdx

```diff
@@ -19,6 +19,10 @@ values={[
 
 ## Heltec Capsule Sensor Rev. 3.0
 
+:::info
```

Comment:
@rcarteraz @pdxlocations what do y'all think about just removing this from the docs altogether? I don't think we need to keep around documentation for something the project no longer supports.

If we do decide to keep it, let's find a way to make this clear without using an admonition (`:::info`)

Reply:
I'm in favor of removing it. Let's not list unsupported hardware.